qcodes/instrument_drivers/QuTech/F1d.py

```diff
@@ -20,6 +20,13 @@ class F1d(Instrument):
             individual modules.
 
         module (int): module number as set on the hardware.
+
+    Parameter:
+        enable_remote: set module in remote mode, switched on the front panel then deactivated
```

Comment:
perhaps you would want to add these lines to the `docstring` argument of the `add_parameter` function? :)